config/testgrids/kubernetes/sig-release/OWNERS

```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- calebamiles
+- justaugustus
+- tpepper
+
+labels:
+- sig/release
```

Comment:
```suggestion
- sig/release
- area/release-eng
```

Reply:
Done, and added this label to config/jobs/kubernetes/sig-release/ as well so the OWNERS files match